scripts/update-repo-internal.sh

```diff
@@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
 case $PARENT in
     centos)
         mkdir -p repo/$PARENT/$RELEASE/packages
-        find package/$DISTRO/ -name *.rpm | \
+        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \
```

Comment:
At the time I had documentation of the config file options in mind. Documenting every last field available for every configurable module would be ideal but it would be an overwhelming task and overkill considering our current manpower and user base. I think just getting the ball rolling would be good for now. Generating a minimal man page would be nice.

[root@15bd60ce98ab lio]# man lstore
No manual entry for lstore

:(

I suppose we should also aspire to have man pages covering the lio_* commands for completeness but you can get a decent usage message from most of them so that lessens the need.

--Matt

----- Original Message -----
From: "Andrew Melo" <notifications@github.com>
To: "accre/lstore-release" <lstore-release@noreply.github.com>
Cc: "Matthew Heller" <matthew.f.heller@accre.vanderbilt.edu>
Sent: Monday, November 9, 2015 12:25:57 AM
Subject: Re: [lstore-release] Rebased version of vandy-cms branch (#24)

> @@ -16,17 +16,20 @@ RELEASE="${DISTRO##*-}"
>  case $PARENT in
>      centos)
>          mkdir -p repo/$PARENT/$RELEASE/packages
> -        find package/$DISTRO/ -name *.rpm | \
> +        find package/$DISTRO/ -name i'*.rpm' | grep -v lstore-release.rpm | \

What do you see us lacking in terms of docs? Simply filling out the README?


-- 
It's dark in this basement.


---
Reply to this email directly or view it on GitHub:
https://github.com/accre/lstore-release/pull/24/files#r44242230